README.md

```
@@ -372,6 +375,76 @@ The system automatically adjusts `edge_blur_amount` based on image analysis:
+ruff check .
+
+# Run flake8 (critical error checks)
+flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+
+# Run bandit (security scan)
+bandit -r .
+```
+
+### Project Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)